docker/test/stress/run.sh

```diff
@@ -193,6 +193,7 @@ stop
 
 # Let's enable S3 storage by default
 export USE_S3_STORAGE_FOR_MERGE_TREE=1
+export $RANDOMIZE_OBJECT_KEY_TYPE=1
```

Comment:
This is incorrect, likely should be `export RANDOMIZE_OBJECT_KEY_TYPE=1`

Reply:
https://github.com/ClickHouse/ClickHouse/pull/58658